Features/WalletConnector/Sources/WalletConnector/Scenes/SignMessageScene.swift

```diff
@@ -56,6 +82,11 @@ public struct SignMessageScene: View {
         .background(Colors.grayBackground)
         .navigationTitle(Localized.SignMessage.title)
         .safariSheet(url: $isPresentingUrl)
+        .sheet(isPresented: $isPresentingFullMessage) {
+            NavigationStack {
+                FullMessageScene(model: model.fullMessageViewModel)
```

Comment:
```suggestion
                FullMessageScene(model: model.fullMessageViewModel)
```

make this scene generic, maybe like `TextMessageScene`